llvm/test/CodeGen/AIE/aie2/schedule/mov_scl.mir

```diff
@@ -167,3 +171,100 @@ body:             |
     $r1 = MOVX_alu_cg 0
     $s0 = MOV_mv_scl $r4
 ...
+
+---
+name:            mova_with_mov_scl_to_R
```

Comment:
Same as previous commit: Please test per port in a file under `resource/port_name.mir` instead of per instruction.